config/prod/EC2-gene-co-expression.yaml

```diff
@@ -0,0 +1,23 @@
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.38/EC2/jc-ec2-linux.j2"
+stack_name: "EC2-gene-co-expression"
+stack_tags:
+  Department: "TBD"
+  Project: "TBD"
+  OwnerEmail: "jake.gockley@sagebase.org"
+  CostCenter: "TBD"
```

Comment:
@jgockley62 please advise on values that should be set in place of `TBD`

cost centers reference: https://github.com/Sage-Bionetworks/aws-infra/tree/master/templates/tags

Reply:
@zaro0508 
```
  Department: "ADTR"
  Project: "ConsensusNetworks"
  OwnerEmail: "jake.gockley@sagebase.org"
  CostCenter: "NIA AMP-AD CC / 101500"
```